js/libs/core.js

```diff
@@ -85,6 +85,10 @@ Fliplet.FormBuilder = (function() {
         return this.showLabel || this.showLabel === undefined;
       };
 
+      component.computed._elementId = function() {
+        return this.name + '-' + this.$parent.$el.dataset.formBuilderId;
```

Comment:
where is `formBuilderId` defined?

Reply:
It is defined in form.js.
<img width="1138" alt="Upplabs #28 answer 1" src="https://user-images.githubusercontent.com/52824207/61210372-f72add00-a704-11e9-9ecb-ffae4f2952a2.png">